deps.ts

```diff
@@ -33,3 +33,10 @@ export {
 export {
   MultipartReader, FormFile
 } from "https://deno.land/std@v0.34.0/mime/multipart.ts";
+
+export { 
+  getCookies,
+  setCookie,
+  delCookie,
+  Cookie
+} from "https://deno.land/std/http/cookie.ts"
```

Comment:
Please use `v0.34.0` deno std. Hopefully in the near future we can make adding these deps in configurable.